[2023-12-12T01:15:39.356+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_stations_data scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-12T01:15:39.360+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_stations_data scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-12T01:15:39.361+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 4
[2023-12-12T01:15:39.366+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_stations_data> on 2023-12-11 00:00:00+00:00
[2023-12-12T01:15:39.371+0000] {standard_task_runner.py:57} INFO - Started process 429 to run task
[2023-12-12T01:15:39.374+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_openaq_catchup', 'load_stations_data', 'scheduled__2023-12-11T00:00:00+00:00', '--job-id', '341', '--raw', '--subdir', 'DAGS_FOLDER/dag_openaq_catchup.py', '--cfg-path', '/tmp/tmp5d7z9o9c']
[2023-12-12T01:15:39.376+0000] {standard_task_runner.py:85} INFO - Job 341: Subtask load_stations_data
[2023-12-12T01:15:39.413+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_openaq_catchup.load_stations_data scheduled__2023-12-11T00:00:00+00:00 [running]> on host 1a187d37aaef
[2023-12-12T01:15:39.460+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_openaq_catchup' AIRFLOW_CTX_TASK_ID='load_stations_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-11T00:00:00+00:00'
[2023-12-12T01:15:39.462+0000] {main.py:22} INFO - Archivo de configuracion leido correctamente.
[2023-12-12T01:15:39.462+0000] {utils.py:25} INFO - Obteniendo datos de https://api.openaq.org/v2/locations...
[2023-12-12T01:15:39.709+0000] {utils.py:73} INFO - Conexi√≥n a la base de datos establecida exitosamente.
[2023-12-12T01:15:41.757+0000] {utils.py:90} INFO - Cargando datos en la tabla stg_locations...
[2023-12-12T01:15:45.558+0000] {utils.py:98} INFO - Datos cargados exitosamente en la tabla stg_locations.
[2023-12-12T01:15:45.564+0000] {main.py:51} INFO - Ejecutando query CALL nicolasmvinciguerra_coderhouse.petl_load_dim_location();...
[2023-12-12T01:15:46.205+0000] {main.py:53} INFO - Se ejecuto correctamente la query: CALL nicolasmvinciguerra_coderhouse.petl_load_dim_location();
[2023-12-12T01:15:46.480+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-12T01:15:46.511+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_openaq_catchup, task_id=load_stations_data, execution_date=20231211T000000, start_date=20231212T011539, end_date=20231212T011546
[2023-12-12T01:15:46.569+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-12T01:15:46.619+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
